test/e2e/specs/editor/various/quote-type-after-inserting.spec.js

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Should this test be just "editor/blocks/quote" (if could include the puppeteer ones we already have later)

Reply:
Makes a lot more sense :+1:

Reply:
https://github.com/WordPress/gutenberg/pull/40466/commits/620abcf1ed64f25f3d72cd7a4da97b08b0a24323